### PR TITLE
Fix #55 - Drop support for Django < 1.8 and Python 2.6.

### LINKED ISSUE #55: Drop support for Django < 1.8
Django 1.7 and older are no longer supported. I propose to drop support for older versions to cleanup the code and ease future development.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: whitenoise/base.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
-from __future__ import absolute_import
```
Comment (luzfcb) [inline whitenoise/base.py]: @timgraham one stupid question that is unrelated to this: some technical inconvenient who takes the need for removal of `from __future__ import absolute_import`

Comment (timgraham) [inline whitenoise/base.py]: At first I incorrectly thought this was only required for Python 2.6, but it's required for 2.7 too. However, at least in Django itself we only include it where necessary so I added back the ones that were required here. I can add them all back if that's preferred, though I don't see an advantage to it.

**Comment by evansd** [comment]
Thanks for this Tim. I'm going to create a separate branch for this Django integration proof-of-concept. As WhiteNoise supports WSGI apps other than Django I'd like to maintain 2.6 compatibility in the standalone project. I know for a fact that I've got users in enterprise environments who are stuck on 2.6.

Obviously for code that's going to be bundled with Django there's no need to maintain compatibility with older Python or Django versions.

Once I've created the new branch do you mind re-submitting your PR against it? (I don't think github lets me change the target branch of existing PRs.)

**Comment by timgraham** [comment]
I wonder if those enterprise users need the latest and greatest version of whitenoise? Of course, it's your project and you can do whatever you like, but I'll leave [a blog post](http://www.snarky.ca/stop-using-python-2-6) in case in might convince you.

**Comment by evansd** [comment]
@timgraham I am now convinced. Thanks for your work on this.

**Pull request merged**